pkgs/applications/networking/cluster/gatekeeper/default.nix

```diff
@@ -2,17 +2,19 @@
 , buildGoModule
 , fetchFromGitHub
 , installShellFiles
+, gatekeeper
+, testers
 }:
 
 buildGoModule rec {
   pname = "gatekeeper";
-  version = "3.12.0";
+  version = "3.13.0";
 
   src = fetchFromGitHub {
     owner = "open-policy-agent";
     repo = "gatekeeper";
     rev = "v${version}";
-    sha256 = "sha256-DGLod05gmDzItPOBaGc2X/oMXSzvH932Wa44R+DUt/U=";
+    sha256 = "sha256-7ciI8KgkC1NRYP8bFW5RnZ2oLXaCuPHQBd2tvgQ+YO8=";
```

Comment:
```suggestion
    hash = "sha256-7ciI8KgkC1NRYP8bFW5RnZ2oLXaCuPHQBd2tvgQ+YO8=";
```

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/249129/commits/73f7d31627796709b9578f52cfa1548a954938c6